earth_enterprise/SConstruct

```diff
@@ -1,6 +1,19 @@
+#
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 #-*- Python -*-
```

Comment:
Nit: Some of these " #-*- Python -*-" lines are above the copyright block and some are below. We should be consistent with these. I don't know if it's worth going through the trouble to change half the files here, but we should pick above or below and try to be consistent with that going forward. I personally think that putting the Python line first looks better because the first like looks bare otherwise.